src/app/portlets/dot-edit-page/layout/dot-edit-layout-designer/dot-edit-layout-designer.component.ts

```diff
@@ -62,8 +63,10 @@ export class DotEditLayoutDesignerComponent implements OnInit {
 
         this.setupLayout();
 
-        if (!this.isLayout()) {
+        if (!this.isLayout() && this.pageState.template.canEdit) {
```

Comment:
I think we need so me more test when the dialog must to be showed or not.
I think we have only have 'should show select edit layout/template dialog'